packages/components/button/src/types.ts

```diff
@@ -14,3 +16,69 @@ export type ButtonStylesProps = {
   isDisabled: boolean;
   isFullWidth: boolean;
 };
+
+interface BaseButtonProps extends CommonProps {
+  children?: React.ReactNode;
+  /**
+   * Determines style variation of Button component
+   * @default secondary
+   */
+  variant?: ButtonVariant;
+  /**
+   * Determines size variation of Button component
+   * @default medium
+   */
+  size?: ButtonSize;
+  /**
+   * Applies active styles
+   * @default false
+   */
+  isActive?: boolean;
+  /**
+   * Disabled interaction and applies disabled styles
+   * @default false
+   */
+  isDisabled?: boolean;
+  /**
+   * Adds dropdown indicator icon
+   */
+  isDropdown?: boolean;
```

Comment:
I'm not sure about this property name... I'd rather make it `withChevron` or something like this. Or do we really need this property at all? Can't we just make a new component - `DropdownButton` that would pass ChevronDown as icon and inherit all other props?

Reply:
Thats a good point, currently this prop only makes it so the right side icon shows up, if we go with the `DropdownButton` we would need to make it so we can align the icon to the right side, since currently it only renders the icon passed on the left side.
But the `withChevron` seems like a good solution too, but I don't have any strong preference over neither of them.

Reply:
We have `alignIcon` property in TextLink that we can bring here.

I prefer the DropdownButton solution more, because by doing this we can make Button package independent from icons package.

Reply:
Would the DropdownButton be a separate package?? Or would we need to make it so Button can accept two different icons to show on the left and right?

Reply:
I think it will be in the dropdown package.